numba/cuda/libdevice.py

```diff
@@ -105,6 +105,7 @@ def core(context, builder, sig, args):
 unarys += [('__nv_lgamma', '__nv_lgammaf', math.lgamma)]
 unarys += [('__nv_sqrt', '__nv_sqrtf', math.sqrt)]
 unarys += [('__nv_log', '__nv_logf', math.log)]
+unarys += [('__nv_log2', '__nv_log2f', math.log2)]
```

Comment:
Now that #6152 is merged, the method for declaring lowerings for unary and binary operations that call libdevice functions is a little different - see e.g. https://github.com/numba/numba/blob/master/numba/cuda/mathimpl.py#L43 - so this should be something like:

```suggestion
unarys += [('log2', 'log2f', math.log2)]
```

Reply:
Thanks, I had an old copy of numba. It should be good now